src/app/pages/login-page/login-page.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { Store } from '@ngrx/store';
+import { IAuthState } from 'src/app/shared/models/interfaces';
+import { setLogin } from 'src/app/shared/store/actions';
+
+@Component({
+  selector: 'app-login-page',
+  templateUrl: './login-page.component.html',
+  styleUrls: ['./login-page.component.scss'],
+})
+export class LoginPageComponent {
+  email = new FormControl('', [Validators.required, Validators.email]);
+  password = new FormControl('', [
+    Validators.required,
+    Validators.minLength(6),
+  ]);
```

Comment:
we can get rid of these variables, just init it within the form and use formControlName